src/main/java/com/rybka/model/ExchangeRequest.java

```diff
@@ -2,9 +2,11 @@
 
 import lombok.AllArgsConstructor;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 
 @Data
 @AllArgsConstructor
+@NoArgsConstructor
```

Comment:
Bad idea to change production code for tests :(

Reply:
fixed